Tasks/Gradle/gradlesonar.ts

```diff
@@ -27,8 +27,8 @@ export function applyEnabledSonarQubeArguments(gradleRun: trm.ToolRunner):trm.To
     var initScriptPath:string = path.join(__dirname, 'sonar.gradle');
 
     // Specify that the build should run the init script
-    gradleRun.arg(['-I', initScriptPath]);
     gradleRun.arg(['sonarqube']);
+    gradleRun.arg(['-I', initScriptPath]);
```

Comment:
@AshwiniChalla - just as an aside, we found that you can pass init Gradle scripts that contain plugins. This way we didn't have to change the user's gradle.build file.